src/models/events.model.js

```diff
@@ -41,9 +42,12 @@ function createModel(app) {
       timestamps: true,
     },
   );
-  event.index({ transactionHash: 1, logIndex: 1, transactionIndex: 1, blockNumber: 1, status: 1 });
```

Comment:
@aminlatifi 
I think it's good to add an index  for status ` event.index({ status: 1 });

Reply:
Index: true was in status field property.

Thanks @mohammadranjbarz.

Reply:
My mistake, you are right